proto-build/src/main.rs

```diff
@@ -147,6 +153,10 @@ fn update_submodules() {
     // be run once.
     run_git(&["submodule", "update", "--init"]);
 
+    info!("Updating osmosis-labs/bech32ibc submodule...");
+    run_git(&["-C", BECH32IBC_DIR, "fetch"]);
+    run_git(&["-C", BECH32IBC_DIR, "reset", "--hard", BECH32IBC_REV]);
```

Comment:
I preffered using libgit2 to submodules where you actually had rust bindings to git. But they ripped it out when 'upstreamed' oh well just griping.

Reply:
:angry: